test-resources/test/test-util/mocked-access-tokens.ts

```diff
@@ -89,7 +89,10 @@ export const subscriberUserPayload = {
   iat,
   iss: subscriberXsuaaUrl,
   zid: TestTenants.SUBSCRIBER,
-  user_id: 'user-sub'
+  user_id: 'user-sub',
+  //Becomes clientId in xssec
+  azp: xsuaaBindingMock.credentials.clientid,
+  aud: [xsuaaBindingMock.credentials.clientid]
```

Comment:
[q] Could you add some context why these are needed? (Maybe not in this PR but in the docs intead...)

Reply:
This is needed when XSSEC validates the JWT, which is mocked in general but for the test along packages I could not mock it with jest. Do you have a place in mind?

Reply:
Maybe add a comment above if it just for testing.

Reply:
extended the comment